swift-blackjack-views/Card.swift

```diff
@@ -0,0 +1,54 @@
+//  Card.swift
+
+import Foundation
+
+class Card {
+    let suit : String
+    let rank : String
+    private(set) var cardLabel = ""
+    private(set) var cardValue: UInt = 0
+    
+    var description: String { return cardLabel }
+    
+    init(suit: String, rank: String) {
+        self.suit = suit
+        self.rank = rank
+        self.cardLabel = "\(suit)\(rank)"
+        self.cardValue = determineCardValue(rank)
+    }
+    
+    class func validSuits() -> [String] {
```

Comment:
There are class-level properties these days... `static let validSuits = [...]`. Feels cleaner in this case than a method